resources/qml/MainWindow/ApplicationMenu.qml

```diff
@@ -91,7 +101,15 @@ Item
         Menu
         {
             id: preferencesMenu
-            title: catalog.i18nc("@title:menu menubar:toplevel", "P&references")
+
+            //On MacOS, don't translate the "Preferences" word.
+            //Qt moves the "preferences" entry to a different place, and if it got renamed can't find it again when it
+            //attempts to delete the item upon closing the application, causing a crash.
+            //In the new location, these items are translated automatically according to the system's language.
+            //For more information, see:
+            //- https://doc.qt.io/qt-5/macos-issues.html#menu-bar
+            //- https://doc.qt.io/qt-5/qmenubar.html#qmenubar-as-a-global-menu-bar
+            title: (Qt.platform.os == "osx") ? "&Preferences" : catalog.i18nc("@title:menu menubar:toplevel", "P&references")
```

Comment:
```suggestion
            title: (Qt.platform.os == "osx") ? "P&references" : catalog.i18nc("@title:menu menubar:toplevel", "P&references")
```
I assume it should be the same in both cases right?

Reply:
It doesn't have to be the same letter in both cases. It's not the same in every language either.